eternalcore-core/src/main/java/com/eternalcode/core/feature/home/command/HomeCommand.java

```diff
@@ -53,6 +59,17 @@ void execute(@Context Player player) {
                     .map(Home::getName)
                     .toList());
 
+            if (this.homesConfig.teleportToDefaultHome) {
+                Optional<Home> mainHome = playerHomes.stream()
+                    .filter(home -> home.getName().equals(this.homesConfig.defaultHomeName))
+                    .findFirst();
+
+                if (mainHome.isPresent()) {
+                    this.homeTeleportService.teleport(player, mainHome.get());
+                    return;
+                }
+            }
+
```

Comment:
please remove Boolean form config and make use of existing firstHome value if possible simplyfing the code